hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/server/UIClientCommWebSocket.java

```diff
@@ -98,6 +98,8 @@ public void uiClientSessionOpen(Session session) {
     public String uiClientMessage(String nameAndJsonStr, Session session) {
         String uiClientId = getUIClientIDFromSession(session);
         MsgLogger.LOG.infoReceivedMessageFromUIClient(uiClientId, session.getId());
+        MsgLogger.LOG.debugf("Received message from UI client [%s] (session [%s]): [%s]", uiClientId, session.getId(),
```

Comment:
@ppalaga  we need to remove this log message - this is going to log the credentials (we scrub the credentials later). We've had a few people complain already that we are logging credentials - so we need to remove this.